vcluster/deploy/control-plane/binary/security/air-gapped.mdx

```diff
@@ -0,0 +1,205 @@
+:::
+
+    <InterpolatedCodeBlock
+      code={`
+wget https://github.com/loft-sh/vcluster/releases/download/v[[GLOBAL:VCLUSTER_VERSION]]/images-private-nodes-optional.txt
+
+./download-images.sh --image-list images-private-nodes-optional.txt --images vcluster-images-optional.tar.gz
+./push-images.sh --registry [[GLOBAL:REGISTRY]] --image-list images-private-nodes-optional.txt
+      `}
+      language="bash"
+      title="Pull, package, and push all optional images"
+    />
+
+  </Step>
+</Flow>
+
+## Configure vCluster {#configure-vCluster}
+
+The vcluster.yaml file contains all configuration settings for your vCluster deployment.
+
+<details>
+  <summary>Use a private registry without credentials</summary>
+
+Set the default private registry that doesn't have authentication.
+
+  <InterpolatedCodeBlock
+    code={`
+controlPlane:
+    advanced:
+        defaultImageRegistry: [[GLOBAL:REGISTRY]]
+      `}
+    language="yaml"
+    title="Setting the private registry"
+  />
+
+</details>
+Before deploying, it's recommended to review the set of configuration options that cannot be updated post deployment. These options require deploying a new vCluster instead of upgrading your vCluster with new options.
+
+### Control-plane options
+
+* [High availability](../high-availability) - Run multiple control plane nodes
+* [CoreDNS](../../../../configure/vcluster-yaml/control-plane/components/coredns) - Currently only CoreDNS deployed by vCluster during startup is supported.
+* Backing Store - Decide how the data of your cluster is stored, must be one of either [embedded SQLite](/vcluster/configure/vcluster-yaml/control-plane/components/backing-store/database/sqlite) (the default) or [embedded etcd](/vcluster/configure/vcluster-yaml/control-plane/components/backing-store/etcd/embedded).
+
+## Deploy vCluster {#deploy-standalone}
+
+After your private registry is populated with the required images, you can proceed with installing the vCluster Standalone.
+
+### Installation
+
+<Flow>
+  <Step>
+  Review your `vcluster.yaml` file.
+
+    <InterpolatedCodeBlock
+      code={`
+controlPlane:
+  advanced:
+    defaultImageRegistry: [[GLOBAL:REGISTRY]]
+  # default vCluster Standalone configuration
+  standalone:
+    enabled: true
+    joinNode:
+      enabled: true  # Optional: Control Plane node will also be considered a worker node
+  privateNodes: # Required for adding additional worker nodes
+  enabled: true
+      `}
+      language="bash"
+      title="Create vcluster.yaml configuration"
+    />
+
+  </Step>
+  <Step>
+  Save the path to the vCluster Standalone Binary and the install script:
+
+  <InterpolatedCodeBlock
+    code={`[[VAR:INSTALL SCRIPT PATH:/tmp/install-standalone.sh]] --vcluster-name standalone --binary [[VAR:BINARY_PATH:/tmp/vcluster]] --skip-download`}
+    language="bash"
+  />
+  </Step>
+</Flow>
```

Comment:
Would be good to add a note that says: For further steps see the primary guide
And link to binary/basics page

Reply:
added